examples/specialty_plots/swotplot.py

```diff
@@ -0,0 +1,180 @@
+"""
+============
+SWOT Diagram
+============
+
+SWOT analysis (or SWOT matrix) is a strategic planning and strategic management
+technique used to help a person or organization identify Strengths, Weaknesses,
+Opportunities, and Threats related to business competition or project planning.
+It is sometimes called situational assessment or situational analysis.
+Additional acronyms using the same components include TOWS and WOTS-UP.
+Source: https://en.wikipedia.org/wiki/SWOT_analysis
+
+"""
+import logging
+from typing import Tuple
+
+import matplotlib.pyplot as plt
+
+_log = logging.getLogger(__name__)
+
+
+def draw_quadrant(q_data: dict, ax: plt.Axes, q_color: str,
+                  q_x: Tuple[float, float, float, float],
+                  q_y: Tuple[float, float, float, float],
+                  q_title_pos: Tuple[float, float] = (0.0, 0.0),
+                  q_title_props: dict = dict(boxstyle='round',
+                                             facecolor="white",
+                                             alpha=1.0),
+                  q_point_props: dict = dict(boxstyle='round',
+                                             facecolor="white",
+                                             alpha=1.0)
+                  ) -> None:
+    """
+    Draw a quadrant of the SWOT plot
+
+    Parameters
+    ----------
+    q_data : dict
+        Data structure passed to populate the quadrant.
+        Format 'key':Tuple(float,float)
+    ax : plt.Axes
+        Matplotlib current axes
+    q_color : str
+        Quadrant color.
+    q_x : Tuple[float]
+        Quadrant X relative coordinates. Format: Tuple(float,float,float,float)
+    q_y : Tuple[float]
+        Quadrant Y relative coordinates. Format: Tuple(float,float,float,float)
+    q_title_pos : Tuple[float,float]
+        Plot title relative position
+    q_title_props : dict, optional
+        Title box style properties. The default is dict(boxstyle='round',
+        facecolor="white", alpha=1.0).
+    q_point_props : dict, optional
+        Point box style properties. The default is dict(boxstyle='round',
+        facecolor="white", alpha=1.0).
+
+    Returns
+    -------
+    None
+
+    """
+    # draw filled rectangle
+    ax.fill(q_x, q_y, color=q_color)
+
+    # draw quadrant title
+    ax.text(x=q_title_pos[0], y=q_title_pos[1],
+            s=str.upper(list(q_data.keys())[0]), fontsize=12, weight='bold',
+            verticalalignment='center', horizontalalignment='center',
+            bbox=q_title_props)
+
+    # fill quadrant with points
+    for k, v in q_data[list(q_data.keys())[0]].items():
+        ax.text(x=v[0], y=v[1], s=k, fontsize=12, weight='bold',
+                verticalalignment='center', horizontalalignment='center',
+                bbox=q_point_props)
+        _log.debug(f"k: {k}, v: {v}")
```

Comment:
Same comment as above about using f-strings in logging. I'm also not sure this message is useful as all it ends up doing is printing the original dictionary in more lines.